### DISCUSSION TITLE
No installed or updates listed.

### DISCUSSION THREAD

**Discussion by ak1dd** [Devolutions/UniGetUI#1761] [Q&A]
Have a machine I hadn't used for over a month with an administrator account and added a default/regular user account (all defaults/Win11/not using MS ID login,only local).

I installed WingetUI under the new non-admin account for All Users.  I ran it and it found about 20 updates.  I let about 5 update but after having to re-authenticate between each install I decided to cancel and try to install as administrator.  I had to press cancel for each install.  I completely closed WingetUI (including the notification area) and restarted.  The UI was invoked but it said zero software updates and zero installed packages.

I tried again right-clicking the WingetUI icon on the desktop (again not running anywhere) and running as administrator. I reboot the machine.  I uninstalled and reinstalled.  None of these produced a  different result.

I read the winget logs and it said "Failed when opening source(s); try the 'source reset' command if the problem persists."  did that - winget source reset --force.  Restarted the App and then restarted the computer.  Didn't work.

I then logged out and logged in as Administrator and the list for both available updates and installed packages were correctly populated.   I logged out and re-logged in as the regular user and nothing is listed.

WINGET SHOW and LIST on the commandline does seem to work under both accounts.

I searched online and github and only found the report on WingetUI github about what to do if packages are missing that is mentioned at the top of the Settings page of the UI.   I did everything in that thread AND uninstalled and reinstalled.   Still nothing listed.

I'm not sure what else to try.  Please find the logs attached.

[winget.log.txt](https://github.com/marticliment/WingetUI/files/14112014/winget.log.txt)
[wingetuilog.txt](https://github.com/marticliment/WingetUI/files/14112015/wingetuilog.txt)

**Comment by marticliment** [comment]
Maybe this helps:
https://github.com/marticliment/WingetUI/issues/1721
